script/tasks/diamondUpdateFacet.sh

```diff
@@ -114,9 +114,9 @@ diamondUpdateFacet() {
           PRIVATE_KEY=$(getPrivateKey $NETWORK $ENVIRONMENT)
           echoDebug "Calculating facet cuts for $SCRIPT..."
           if [[ $NETWORK == "zksync" ]]; then
-            RAW_RETURN_DATA=$(docker run --rm -it --volume .:/foundry -u $(id -u):$(id -g) -e FOUNDRY_PROFILE=zksync -e NO_BROADCAST=true -e NETWORK=$NETWORK -e FILE_SUFFIX=$FILE_SUFFIX -e USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND -e PRIVATE_KEY=$PRIVATE_KEY foundry-zksync forge script "$UPDATE_SCRIPT" -f $NETWORK -vvvv --json --silent --skip-simulation --legacy --slow --zksync)
+            RAW_RETURN_DATA=$(docker run --rm -it --volume .:/foundry -u $(id -u):$(id -g) -e FOUNDRY_PROFILE=zksync -e NO_BROADCAST=true -e NETWORK=$NETWORK -e FILE_SUFFIX=$FILE_SUFFIX -e USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND -e PRIVATE_KEY=$PRIVATE_KEY foundry-zksync forge script "$UPDATE_SCRIPT" -f $NETWORK --json --skip-simulation --legacy --slow --zksync)
```

Comment:
Why `-vvvv` is removed from here. Won't it be nice to have for debugging purposes?

Reply:
If you could describe why it was needed

Reply:
The latest foundry version was complaining that certain flags cannot be combined. But I think you are right that it was unnecessary to remove both. 
The ideal combination should be --json and -vvvv so we have maximum output in JSON format. Thanks for spotting and pointing this out. Will update now

Reply:
will actually move this topic in a separate PR that we can merge before this one

Reply:
update: 
the output is the same, with or without -vvvv flag. 
However, it's better to be explicit so I will add it and add a v for max verbosity. PR is ready for review:
https://github.com/lifinance/contracts/pull/965

Reply:
outdated after the merged PR

Reply:
thanks for update!